cmd/node/main.go

```diff
@@ -86,62 +86,70 @@ VERSION:
 
 	// genesisFile defines a flag for the path of the bootstrapping file.
 	genesisFile = cli.StringFlag{
-		Name:  "genesis-file",
-		Usage: "The node will extract bootstrapping info from the genesis.json",
+		Name: "genesis-file",
+		Usage: "The `filepath` for the genesis file in JSON format. This file should contain initial data to bootstrap" +
+			" from, such as initial balances for accounts.",
 		Value: "./config/genesis.json",
 	}
 	// nodesFile defines a flag for the path of the initial nodes file.
 	nodesFile = cli.StringFlag{
-		Name:  "nodes-setup-file",
-		Usage: "The node will extract initial nodes info from the nodesSetup.json",
+		Name: "nodes-setup-file",
+		Usage: "The `filepath` for the nodes setup in JSON format. This file should contain initial nodes info, such as" +
+			" consensus group size, round duration, validators public keys and so on",
 		Value: "./config/nodesSetup.json",
 	}
 	// sk defines a flag for the path of the multi sign private key used when starting the node
 	sk = cli.StringFlag{
 		Name:  "sk",
-		Usage: "Private key that the node will load on startup and will sign blocks",
+		Usage: "The `filepath` for the PEM file which includes the private key the node will use for block signing",
 		Value: "",
 	}
 	// configurationFile defines a flag for the path to the main toml configuration file
 	configurationFile = cli.StringFlag{
-		Name:  "config",
-		Usage: "The main configuration file to load",
+		Name: "config",
+		Usage: "The `filepath` for the main configuration file in TOML format. This file should contain the main" +
+			" configurations such as storage setups, epoch duration and so on",
 		Value: "./config/config.toml",
 	}
 	// configurationEconomicsFile defines a flag for the path to the economics toml configuration file
 	configurationEconomicsFile = cli.StringFlag{
-		Name:  "config-economics",
-		Usage: "The economics configuration file to load",
+		Name: "config-economics",
+		Usage: "The `filepath` for the economics configuration file in TOML format. This file should contain" +
+			" economics configurations such as minimum gas price for a transactions and so on",
 		Value: "./config/economics.toml",
 	}
 	// configurationPreferencesFile defines a flag for the path to the preferences toml configuration file
 	configurationPreferencesFile = cli.StringFlag{
-		Name:  "config-preferences",
-		Usage: "The preferences configuration file to load",
+		Name: "config-preferences",
+		Usage: "The `filepath` for the preferences configuration file in TOML format. This file should contain" +
+			" preferences configurations, such as the node display name or the shard to start in when starting as observer",
```

Comment:
The path to the preferences file. This TOML file contains configuration such as ...